### PR TITLE
don't send a notification when expense approved to host if collective hasn't been approved

### PR DESCRIPTION
**Pull request by xdamman**
Fixes https://github.com/opencollective/opencollective/issues/1967

### CODE REVIEW

File: server/models/Expense.js [hunk 1]
```diff
@@ -194,7 +194,7 @@ export default function(Sequelize, DataTypes) {
       CollectiveId: this.collective.id,
       data: {
         host: get(host, 'minimal'),
-        collective: this.collective.minimal,
+        collective: { ...this.collective.minimal, isActive: this.collective.isActive },
```
Comment (znarf) [inline server/models/Expense.js]: can we add `isActive` to minimal instead?

Comment (xdamman) [reply server/models/Expense.js]: Do we really want to "pollute" the `minimal` used elsewhere with this information that is only needed here?

Comment (znarf) [reply server/models/Expense.js]: I don't know the history behind it. Looks weird to extend it with just one property.

Now that I'm looking at it, I'm wondering why the following are `minimal`:

```
twitterHandle,
githubHandle,
publicUr,
mission,
isSupercollective,
```

Comment (xdamman) [reply server/models/Expense.js]: This entire thing is bogus. It was initially meant for storing a minimum set of data for the activity table. But we should revisit that at some point.

**Pull request merged**